tests/test_config_registry_and_test_execution.py

```diff
@@ -69,10 +71,7 @@ def test_test_execution_with_pattern(tmp_test, capsys):
     exit_code = test_execution.execute(cfg, pattern_inject="my_test.py")
     ConfigRegistry.delete_aux_con()
 
-    output = capsys.readouterr()
-    assert "FAIL" not in output.err
-    assert "Ran 0 tests" in output.err
-    assert exit_code == test_execution.ExitCode.ALL_TESTS_SUCCEEDED
+    assert exit_code == test_execution.ExitCode.NO_TEST_SELECTED
```

Comment:
Why do we expect the exit code to be NO_TEST_SELECTED if we're not using it anywhere and why does the test pass? Also, maybe it can be removed in that case?